di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/models/JarUserInfo.java

```diff
@@ -8,8 +8,14 @@ public record JarUserInfo(
                 Essential coreIdentityJwtClaim,
         @JsonProperty(value = "https://vocab.account.gov.uk/v1/address") Essential addressClaim,
         @JsonProperty(value = "https://vocab.account.gov.uk/v1/passport") Essential passportClaim,
-        @JsonProperty(value = "https://vocab.account.gov.uk/v1/socialSecurityNumber")
+        @JsonProperty(value = "https://vocab.account.gov.uk/v1/socialSecurityRecord")
                 Essential ninoClaim,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+                @JsonProperty(value = "https://vocab.account.gov.uk/v1/drivingPermit")
+                Essential drivingPermitClaim,
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+                @JsonProperty(value = "https://vocab.account.gov.uk/v1/returnCode")
+                Essential returnCodeClaim,
```

Comment:
`{"essential": true}` and `null` are both valid values for these, which makes this a little tricky. See https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter for details